EpsilonCCS/Inc/CanParsers/BmsCanParser.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "stm32f4xx.h"
+
+void parseBmuCanMessage(uint32_t stdId, uint8_t data[8]);
```

Comment:
I will just leave you with the great Linus Torvalds on passing arrays into functions. https://lkml.org/lkml/2015/9/3/428